nextflow.config

```diff
@@ -62,6 +62,8 @@ params {
   umi = null // no umi
   read_structure1 = null // no umi
   read_structure2 = null // no umi
+  model_type = 'WGS'  // For DeepVariant
```

Comment:
These parameters are specific to deep variant? Is there a nextflow convention to check for parameter inputs?

Reply:
Yeah, nf-core has JSON schemas with each pipeline. I've updated the one for Sarek to account for the new parameter.